crates/accelerate/src/target_transpiler/mod.rs

```diff
@@ -368,21 +374,21 @@ impl Target {
         qargs: Option<Qargs>,
         properties: Option<InstructionProperties>,
     ) -> PyResult<()> {
+        let qargs_extract = qargs.as_ref().cloned().unwrap_or_default();
```

Comment:
Why do we need to clone here? We didn't have that before

Reply:
This was a mistake in my end, this can probably be modified to be something along the lines of:
```suggestion
        let qargs_extract = qargs.as_ref().unwrap_or(&[]);
```

Reply:
Fixed in 017f657